pom.xml

```diff
@@ -82,6 +82,13 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>2.5.3</version>
+        <inherited>true</inherited>
```

Comment:
If this is inherited, where is it inheriting it from?  You probably want to set up some of the other headers here.  Take a look at what the Savoir parent project does here:

https://github.com/savoirtech/savoir-parent/blob/master/pom.xml

We use this all over the place and it works well for us.  You can customize imports/exports and stuff using the osgi.* properties.